src/content/community/releases/_index.en.md

```diff
@@ -4,6 +4,27 @@ title = "Releases"
 weight = 30
 +++
 
+## v0.8.0
+
+* Added support for connecting clusters that use the OVN (ovn-kubernetes) CNI plugin.
+* The active Gateway now performs periodic health checks on the connections to remote clusters,updates the Gateway
+  connection status, and adds latency statistics.
+* Gateways now export the following connection metrics which can be used with Prometheus. These are currently only supported
+  for the Libreswan cable driver.
+  * The count of bytes transmitted and received between Gateways.
+  * The number of connections between Gateways and their corresponding status.
+  * The timestamp of the last successful connection established between Gateways.
+  * The RTT latency between Gateways.
+* The Libreswan cable driver is now the default.
+* The strongSwan cable driver is deprecated and will be removed in a future release.
+* Lighthouse now prefers a Service in the local cluster before choosing a remote cluster round-robin.
```

Comment:
This one is not accurate; If an exported Service is available in the local cluster, Lighthouse DNS *always* returns the IP address of the local ClusterIP Service. If multiple clusters export a Service with the same name and from the same namespace, Lighthouse DNS load-balances between the clusters in a round-robin fashion.

Reply:
I believe that's what my statement says unless I'm missing something. Specifically how is it inaccurate wrt to your?

Reply:
If a Service is available in the local cluster, it always going to serve Pods from the local cluster. The current statement is a bit confusing IMO, because one can read it as "first request is going to be handled by the local cluster, following requests will be served as round-robin".

Reply:
yeah that's not the intended meaning of course but I can  see how someone could interpret it that way although it doesn't really make much sense 😄 I'll clarify as you suggest.